lookout/style/format/rules.py

```diff
@@ -57,11 +56,16 @@ class Rules(BaseEstimator, ClassifierMixin):
 
     CompiledRulesType = Dict[int, CompiledFeatureRules]
 
+    BottomUpGreedyBudget = NamedTuple("BottomUpGreedyBudget", (
```

Comment:
I see that the algorithm's name is "top-down", so makes sense to rename this to `TopDownGreedyBudget`